### ISSUE TITLE
How to choose the specific model from 40 models

### ISSUE THREAD

**Issue by etudemin** [JiayuYANG/CVP-MVSNet#13]
Hi @JiayuYANG,

Thanks for your great work!

In this project, authors provided the pre-trained model of the 27th epoch (model_0027.ckpt). Would you kindly explain how the authors choose the specific model from 40 models?

I've retrained your network using the default setting, and got 40 models which correspond to 40 epochs. Attached is the generated log file, and the epoch loss is dropping gradually during 40 epochs. Seems that there is no validation process during training, so it's a little confusing about how to choose the specific model to test.

Thanks for your kind reply in advance, and please correct me if I miss something.
[20200831-0132.log](https://github.com/JiayuYANG/CVP-MVSNet/files/5212748/20200831-0132.log)

**Comment by JiayuYANG** [comment]
Hi @etudemin ,

You can modify my `eval.sh` to use the validation set provided by [MVSNet](https://github.com/YoYo000/MVSNet) to calculate depth errors if you would like to do validation on depth maps.

And after that you can also use `fusion.sh` to generate validation point clouds and compute the accuracy and completeness scores using [DTU evaluation code](http://roboimagedata.compute.dtu.dk/?page_id=36) if you would like to do validation on point clouds.

Cheers,
Jiayu

**Comment by etudemin** [comment]
Hi @JiayuYANG ,

I will go check the validation set. Thank you very much for your suggestions!

Sincerely,
etudemin
